pom.xml

```diff
@@ -731,6 +814,9 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-javadoc-plugin</artifactId>
                     <version>3.1.1</version>
+                    <configuration>
+                        <doclint>all</doclint>
```

Comment:
What is this for?

Reply:
Originally `doclint` is enabled via the [`maven-compiler-plugin` option here](https://github.com/feast-dev/feast/pull/946/files#diff-600376dffeb79835ede4a0b285078036L712). This enables `doclint` from `maven-javadoc-plugin` so that we can disable doclint for classpath of the OpenAPI client which fails `doclint`.

Reply:
But there is an exclusion. Why did this work in the past but not any more?

Reply:
I'm am not entirely sure myself.
The effect of this move is that doclint moves from the compile maven lifecycle to the test maven lifecycle.
Doclint still works and flags issues with the docs.